pages/index.tsx

```diff
@@ -23,16 +22,6 @@ const Home = () => {
     console.log("👋 Welcome to Replay!");
   }
 
-  const handleClick = (step: number) => {
-    setCurrentStep(step);
-    const newCheckedItems = [...checkedItems];
-    newCheckedItems[step - 1] = true;
-    setCheckedItems(newCheckedItems);
-
-    const allStepsCompleted = newCheckedItems.every((item) => item);
-    setAnimateSuccess(allStepsCompleted);
-  };
-
```

Comment:
this is a big no no
https://www.loom.com/share/bbdab8ade88542199c450b7ffaa6a099